cyto_dl/train.py

```diff
@@ -96,23 +96,24 @@ def train(cfg: DictConfig, data=None) -> Tuple[dict, dict]:
     if cfg.get("train"):
         log.info("Starting training!")
 
-        if cfg.get("weights_only"):
-            assert cfg.get(
+        load_params = cfg.get("checkpoint")
+        if load_params.get("weights_only"):
```

Comment:
what's the difference between loading the state dict to the model and setting ckpt_path to None vs just passing in ckpt_path to the trainer?

Reply:
passing the ckpt_path to the trainer also loads non-state dict optimization metadata (and always does a strict load). For the plugin we want to e.g. let people finetune megaseg, so we want to only load the weights to let people continue training past the original models `trainer.max_epochs`

Reply:
oh gotcha, maybe just add exactly this as a comment there?